docs/integrations/prefect-aws/ecs_guide.mdx

```diff
@@ -318,7 +318,7 @@ push:
 
  # the deployments section allows you to provide configuration for deploying flows
 deployments:
-- name: my_ecs_deployment
+    - name: my_ecs_deployment
```

Comment:
@EmilRex Im pretty sure these are both valid right? and all of the current docs examples use the [current](https://docs.prefect.io/v3/deploy/infrastructure-concepts/prefect-yaml) format